grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJob.groovy

```diff
@@ -186,13 +187,14 @@ class ClientBatchJob {
 
         ClientBatchJob build() {
             final jobParameters = [
-                "timestamp"          : System.currentTimeMillis() as String,
-                "${PATH}"            : pathConfiguration.path,
-                "${HOST}"            : serverBuilder.host,
-                "${PORT}"            : serverBuilder.port,
-                "${USERNAME}"        : credentialsBuilder.username,
-                "${PASSWORD}"        : credentialsBuilder.password,
-                "${WORKFLOW_CONFIGS}": pathConfiguration.workflowConfigIds.join("|")
+                    "timestamp"           : System.currentTimeMillis() as String,
+                    "${PATH}"             : pathConfiguration.path,
+                    "${HOST}"             : serverBuilder.host,
+                    "${PORT}"             : serverBuilder.port,
+                    "${USERNAME}"         : credentialsBuilder.username,
+                    "${PASSWORD}"         : credentialsBuilder.password,
+                    "${EXCLUDE_PATHS}"    : pathConfiguration.excludePaths.join("<sp>"),
```

Comment:
The problem with `|` is that it's valid to be in a path (whereas it's not valid to be in a workflow config id, so it can be used as a delimiter for those). Of course, so can `<sp>`, but that gets to [my other comment..](https://github.com/TWCable/grabbit/pull/29#discussion_r31847833)